### ISSUE TITLE
AppDaemon installation fails

### ISSUE THREAD

**Issue by viktak** [AppDaemon/appdaemon#891]
I am trying to install the latest appdaemon on a linux server, using docker following the installation steps [here](https://appdaemon.readthedocs.io/en/stable/DOCKER_TUTORIAL.html#testing-your-system).

This is the exact command I run:
`docker run --rm -it -p 5050:5050 -e HA_URL="http://192.168.1.99:8123" -e TOKEN="eyJ0eXAiOiJKV  - more of this token... -  ZgdG8D905_P-PllYIc" -e DASH_URL="http://$HOSTNAME:5050" acockburn/appdaemon:latest`

According to the documentation, I should get messages like "Connected to Home Assistant ", "Hello", etc, but I don't get any of them. Instead this is all I can see:

> Unable to find image 'acockburn/appdaemon:latest' locally
latest: Pulling from acockburn/appdaemon
c9b1b535fdd9: Already exists 
2cc5ad85d9ab: Pull complete 
61614c1a5710: Pull complete 
0522d30cde10: Pull complete 
407da27a0336: Pull complete 
d33c813a35a8: Pull complete 
2df549ea0f84: Pull complete 
de1f85ae0a47: Pull complete 
d500014351bc: Pull complete 
da4a57f8075d: Pull complete 
d0bb0988f896: Pull complete 
Digest: sha256:a1a86da583e14f6b779d2c330e6eb5115c8b8c8d7a55de02b4c7f3a978152c87
Status: Downloaded newer image for acockburn/appdaemon:latest
2020-02-19 14:04:55.998462 INFO AppDaemon: AppDaemon Version 4.0.1 starting
2020-02-19 14:04:55.999290 INFO AppDaemon: Python version is 3.8.1
2020-02-19 14:04:55.999845 INFO AppDaemon: Configuration read from: /conf/appdaemon.yaml
2020-02-19 14:04:56.000253 INFO AppDaemon: Added log: AppDaemon
2020-02-19 14:04:56.000515 INFO AppDaemon: Added log: Error
2020-02-19 14:04:56.000719 INFO AppDaemon: Added log: Access
2020-02-19 14:04:56.000914 INFO AppDaemon: Added log: Diag
2020-02-19 14:04:56.001127 ERROR AppDaemon: time_zone not specified in appdaemon.cfg
2020-02-19 14:04:56.001309 ERROR AppDaemon: latitude not specified in appdaemon.cfg
2020-02-19 14:04:56.001487 ERROR AppDaemon: longitude not specified in appdaemon.cfg
2020-02-19 14:04:56.001656 ERROR AppDaemon: elevation not specified in appdaemon.cfg

I wonder if the current documentation is is relevant to the actual version.

Any idea how to go on?

**Comment by dlashua** [comment]
I don't believe the docker image auto builds configuration correctly like it did for v3. So we'll look in to that.

In the mean time, you're going to want persistent configuration anyway. So, I recommend this:
```
docker run --name=appdaemon -d -p 5050:5050 \
  --restart=always \
  -v <your_conf_folder>:/conf \
  acockburn/appdaemon:latest
```
And, before running that, in your_conf_folder, include an appdaemon.yaml file that conforms with the docs for AD v4 (which includes specifying latitude, longitude, etc). 
I recommend using that Docker command line

**Comment by viktak** [comment]
Hmmm, Yesterday I have tried specifying a config folder just as you proposed, I just didn't create the appdaemon.yaml file, so it failed the same way.
Now I cannot try creating the appdaemon.yaml file first as in the meantime I succeeded through Supervisor's Add-on store. But soon I will test it in another environment.
I did notice that when I finally created the appdaemon docker (through the Add-on store) an appdaemon.yaml file was indeed created for me with the correct coordinates.
